### DISCUSSION TITLE
Brotli

### DISCUSSION THREAD

**Discussion by BlackAsLight** [oakserver/oak#654] [Q&A] [answered]
I see that Oak supports sending Brotli type files under the `Context.send` function, and although I'm not serving any `.br` files, my browser seems to be receiving the files with `Content-Encoding: br`. Is this Oak's doing this Brotli conversion on the fly or is something else going on? When I provide gzip versions of my static files the `Content-Encoding` does seem to change to `gz`.

**Comment by kitsonk** [comment, marked as answer]
No, it is Deno. See: https://docs.deno.com/deploy/api/compression

The `.send()` will use compressed versions if configured to do so, as you have noticed, and Deno will then respect that.

**Reply by BlackAsLight** [reply]
The link says that is a case for Deno Deploy, but I'm not using Deploy. Is Deno still just doing it anyway?

**Reply by kitsonk** [reply]
Yes.
